### DISCUSSION TITLE
Problem trying to retrieve the json API from the Play Store console

### DISCUSSION THREAD

**Discussion by omensight** [fastlane/fastlane#21550] [Q&A]
Hello, I'm getting this error when I try to access the API page in developer console, I'm following the guide provided at the moment of initialization of Fastlane.
![image](https://github.com/fastlane/fastlane/assets/34387536/41e2064d-49f3-4e44-a658-55a21de485a3)
Thanks.

**Comment by sofi-kry** [comment]
We are experiencing the same issue. We had everything set up and it was working until [this happened](https://github.com/fastlane/fastlane/issues/21537). Given than we got no answer we tried to set things up again, only to find that the way to do it is different than before. The problem now is that the [documentation](https://docs.fastlane.tools/getting-started/android/setup/) explains how to do it the old way. Please help 🙏

**Reply by omensight** [reply]
> We are experiencing the same issue. We had everything set up and it was working until [this happened](https://github.com/fastlane/fastlane/issues/21537). Given than we got no answer we tried to set things up again, only to find that the way to do it is different than before. The problem now is that the [documentation](https://docs.fastlane.tools/getting-started/android/setup/) explains how to do it the old way. Please help 🙏 

I will publish a guide when I'm out of work.
